### DISCUSSION TITLE
[Announment] Welcome new committer: Wei Peng

### DISCUSSION THREAD

**Discussion by nobodyiam** [apolloconfig/apollo#3734] [Announcements]
Hi Community,

On behalf of the Apollo PMC, I am pleased to announce that Wei Peng a.k.a pengweiqhca, has been voted in as Apollo committer.

Please join me to say congratulations to him!

@pengweiqhca, would you please briefly introduce yourself to the community?

Best Regards,
Jason

**Comment by kezhenxu94** [comment]
Welcome @pengweiqhca !! Congratulations

**Reply by pengweiqhca** [reply]
Thanks.

**Comment by lepdou** [comment]
Welcome @pengweiqhca  Congratulations~~~~
